services/issue/issue.go

```diff
@@ -125,6 +125,17 @@ func UpdateAssignees(issue *models.Issue, oneAssignee string, multipleAssignees
 	return
 }
 
+// DeleteIssue deletes an issue
+func DeleteIssue(doer *user_model.User, issue *models.Issue) error {
+	if err := models.DeleteIssue(issue); err != nil {
+		return err
+	}
+
+	// notification.NotifyDeleteIssue(doer, issue)
```

Comment:
Does it make sense to send a notification for completely removing an issue? If so, even per E-Mail (without the original content of course)? I'll have to look at the notification system later, because I already had enough codebase complexity for now. :wink:

Reply:
You mean I should not (or do not have to) implement notifications in this PR, but prepare it?